pages/windows/mount.md

```diff
@@ -0,0 +1,32 @@
+# mount
+
+> Mount Network File System (NFS) network shares.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/mount>.
+
+- Mount a share to the "Z" drive letter:
+
+`mount \\{{computer_name}}\{{share_name}} {{Z:}}`
+
+- Mount a share to the next available drive letter:
+
+`mount \\{{computer_name}}\{{share_name}} *`
+
+- Mount a share with a timeout in seconds:
```

Comment:
What exactly is this timeout for?

Reply:
The connection to the share. 🤔

Reply:
The official document shows:

>  Sets the time-out value in seconds for a remote procedure call (RPC). Acceptable values are 0.8, 0.9, and any integer in the range 1-60; the default is 0.8.

Should we mention the value range here?

Reply:
Probably would be good. 👍 Not sure on the wording though.

Reply:
```suggestion
- Mount a share with a read timeout in seconds (defaults to 0.8, can be 0.9 or 1 to 60):
```

Reply:
Thanks @sbrl!